salt/modules/yumpkg.py

```diff
@@ -924,6 +925,35 @@ def list_upgrades(refresh=True, **kwargs):
 list_updates = salt.utils.alias_function(list_upgrades, 'list_updates')
 
 
+def list_downloaded():
+    '''
+    .. versionadded:: Nitrogen
```

Comment:
As this is still not merged, I would actually say to set this as `.. versionadded:: 2017.7.0` (the rest of the modules in this branch have been renamed in this PR: https://github.com/saltstack/salt/pull/41746). I should have suggested this from the first time, sorry.